shell/platform/android/io/flutter/view/FlutterNativeView.java

```diff
@@ -20,14 +20,21 @@
 import io.flutter.embedding.engine.dart.PlatformMessageHandler;
 
 public class FlutterNativeView implements BinaryMessenger {
+
+    interface NativeViewHelper extends FlutterPluginRegistry.ViewRegistry {
```

Comment:
Let's avoid calling anything a "helper".  That term doesn't inform anyone about what the thing does.  If it has no discernible role then we should re-evaluate the interface.

Reply:
I am happy to rename any of these interfaces to your liking :-)